src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -18,7 +18,9 @@ namespace ts.codefix {
 
             switch (token.kind) {
                 case ts.SyntaxKind.Identifier:
-                    return deleteIdentifier();
+                    let actions = deleteIdentifier(<Identifier>token);
+                    (actions || (actions = [])).push(prefixIdentifierWithUnderscore(<Identifier>token));

```

Comment:
we should not do this all the time. only for parameters and for..of/for .. in variables